test/unit/lib/addAddressLine.js

```diff
@@ -0,0 +1,118 @@
+const chai = require('chai');
+
+const addAddressLine = require('../../../app/lib/addAddressLine');
+
+const expect = chai.expect;
+
+function getValidServices() {
+  const nameOne = 'place name one';
+  const nameTwo = 'place name two';
+  const address = {
+    city: 'city',
+    county: 'county',
+    line1: 'line1',
+    line2: 'line2',
+    line3: 'line3',
+    postcode: 'AB12 3CD',
+  };
+  const services = [{
+    address,
+    name: nameOne,
+  },
+  {
+    address,
+    name: nameTwo,
+  }];
+
+  return services;
+}
+
+describe('addAddressLine', () => {
+  it(
```

Comment:
All of the text descriptions for the `it` blocks start on the following line but the lines don't look long enough to fail due to length. Is there another reason why they are formatted this way? If not, can they be brought back to a single line?

Reply:
Overzealous code formatter, have improved it now, thanks.